### PR TITLE
ADD dc motor code

### PR DESCRIPTION
**Pull request by mrfrozen97**

### CODE REVIEW

**Comment by mrfrozen97** [comment]
In case....... I am Tanishq

**Comment by aditya-c137** [comment]
Nice job tanishq... I won't accept the merge request right away, let others work on their code... Work on Stepper motors next... React with an emoji or replay with something so i know that you have read this

**Comment by aditya-c137** [comment]
Also go through this [link](https://chris.beams.io/posts/git-commit/) to write effective commit messages, no pressure you can take your time with this.

**Comment by mrfrozen97** [comment]
Stepper motor code

File: STM/Sensor/Tanishq/stepper_motor.ino [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+void loop() {
+
+
+  //  Taking input for the speed of rotation..........................................
+  
+  Serial.print("Enter the steps you want per second (0-");
+  Serial.print(MOTOR_STEPS);
+  rot_speed = Serial.parseInt();
+  if(rot_speed>=0 && rot_speed<=MOTOR_STEPS){                                               //check if speed is not any unusual value. if so tell it to use
```
Comment (aditya-c137) [inline STM/Sensor/Tanishq/stepper_motor.ino:37]: Can you refactor this if block? Is there a better way to write this piece of code?

File: STM/Sensor/Tanishq/stepper_motor.ino [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      duration = 0;
+      Serial.println("Duration can't be negative");
+      }
+
+
+
+     //Taking input for the direction of the motor........................................
+    Serial.println("Enter the direction(for clockwise enter 1 and for anticlockwise enter 1) : ");
+    direction_motor = Serial.parseInt();
+    if(direction_motor==0 || direction_motor==1){
```
Comment (aditya-c137) [inline STM/Sensor/Tanishq/stepper_motor.ino:61]: Same with this if block

**Review by aditya-c137 — CHANGES_REQUESTED** [review]

**Comment by aditya-c137** [comment]
You can work on servo after refactoring.

File: STM/Sensor/Tanishq/stepper_motor.ino (hunk 1 above)
Comment (mrfrozen97) [reply STM/Sensor/Tanishq/stepper_motor.ino:37]: I didn't understand what I need to do.
Is it while(Serial.avaliable==0){}

**Pull request merged**